stockAndPrices/services/app/main.py

```diff
@@ -1,7 +1,12 @@
 from flask import Flask, request
+from flask_cors import CORS
+from configuration.Origin import Origin
 
 
 app = Flask(__name__)
+app.config.from_pyfile("configuration/DevConfig.py")
```

Comment:
Can you change this to configuration/Config.py?

Reply:
Updated